### PR TITLE
Ft auto image update

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- [x] Auto updates container images
- [x] [use environment variables for container registry](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/0618ee39f85fdbef08c48cd2d26515a4a476b929)
- [x] [redis chart](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/f3415d98685d475ad4d78a7135dca481ca342684)
- [x] [trigger all workflows](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/338dd7d8fc7f704744eacfe034b51a6ea61fda16)
- [x] [use the kafka PROD connector in production](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/ca58771bfead5adf55788933aec52ae5759582c6)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**
- n/a

**_HOW DO I TEST OUT THIS PR?_**
- Do a static code analysis

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

File: .github/workflows/deploy-apis-to-production.yml [hunk 1]
```diff
@@ -350,9 +355,9 @@ jobs:
 
       - name: Deploy to k8s cluster
         run: |
-          export MEASUREMENTS_CONNECT_IMAGE=${{ secrets.REGISTRY_URL }}/${{ secrets.PROJECT_ID }}/measurements-connect:${{ needs.image-tag.outputs.build_id }}
-          export STAGE_KAFKA_CLUSTER=${{ secrets.STAGE_KAFKA_CLUSTER }}
-          cat k8s/kafka/connectors/measurements-connect.yaml | sed "s/{{MEASUREMENTS_CONNECT_IMAGE}}/$MEASUREMENTS_CONNECT_IMAGE/g" | sed "s/{{KAFKA_CLUSTER}}/$STAGE_KAFKA_CLUSTER/g" | kubectl apply -n message-broker -f-
+          export MEASUREMENTS_CONNECT_IMAGE=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/measurements-connect:${{ needs.image-tag.outputs.build_id }}
+          export PROD_KAFKA_CLUSTER=${{ secrets.PROD_KAFKA_CLUSTER }}
+          cat k8s/kafka/connectors/measurements-connect.yaml | sed "s/{{MEASUREMENTS_CONNECT_IMAGE}}/$MEASUREMENTS_CONNECT_IMAGE/g" | sed "s/{{KAFKA_CLUSTER}}/$PROD_KAFKA_CLUSTER/g" | kubectl apply -n message-broker -f-
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:360]: [use the kafka PROD connector in production](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/ca58771bfead5adf55788933aec52ae5759582c6)

File: .github/workflows/deploy-apis-to-production.yml [hunk 2]
```diff
@@ -7,6 +7,11 @@ on:
     types:
       - closed
 
+env:
+  REGISTRY_URL: gcr.io
+  PROJECT_ID: airqo-250220
+  DEPLOY_BRANCH: deployment/argocd
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:13]: [use environment variables for container registry](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/0618ee39f85fdbef08c48cd2d26515a4a476b929)

File: workflow-trigger [hunk 3]
```diff
@@ -1 +1 @@
-2022-10-20T20:00:00Z
+2022-11-28T20:00:00Z
```
Comment (123MwanjeMike) [inline workflow-trigger:1]: [trigger all workflows](https://github.com/airqo-platform/AirQo-api/pull/1205/commits/338dd7d8fc7f704744eacfe034b51a6ea61fda16)

File: k8s/meta-data/values-prod.yaml [hunk 4]
```diff
@@ -17,7 +17,7 @@ podAnnotations: {}
 
 resources:
   limits:
-    cpu: 500m
+    cpu: 1000m
```
Comment (123MwanjeMike) [inline k8s/meta-data/values-prod.yaml:20]: Allocate meta data api more resources

File: .github/workflows/deploy-apis-to-staging.yml [hunk 5]
```diff
@@ -1098,8 +1233,20 @@ jobs:
...
+
+      - name: Update corresponding helm values file
+        uses: fjogeleit/yaml-update-action@main
+        with:
+          valueFile: "k8s/gp-model/values-stage.yaml"
+          propertyPath: "image.tag"
+          value: ${{ needs.image-tag.outputs.build_id }}
+          updateFile: true
+          branch: ${{ env.DEPLOY_BRANCH }}
+          message: "Update gp-model staging image tag to ${{ needs.image-tag.outputs.build_id }}"
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:1249]: Auto image update

**Review by 123MwanjeMike — COMMENTED** [review]
Guiding comments

**Review by Baalmart — APPROVED** [review]
This LGTM, thanks @123MwanjeMike

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**